test/parallel/test-repl-persistent-history.js

```diff
@@ -53,20 +55,38 @@ const ENTER = { name: 'enter' };
 const CLEAR = { ctrl: true, name: 'u' };
 // Common message bits
 const prompt = '> ';
-const replDisabled = '\nPersistent history support disabled. Set the ' +
-                     'NODE_REPL_HISTORY environment\nvariable to a valid, ' +
-                     'user-writable path to enable.\n';
-const convertMsg = '\nConverting old JSON repl history to line-separated ' +
-                   'history.\nThe new repl history file can be found at ' +
-                   path.join(common.tmpDir, '.node_repl_history') + '.\n';
-const homedirErr = '\nError: Could not get the home directory.\n' +
-                   'REPL session history will not be persisted.\n';
-const replFailedRead = '\nError: Could not open history file.\n' +
-                       'REPL session history will not be persisted.\n';
-const sameHistoryFilePaths = '\nThe old repl history file has the same name ' +
-                             'and location as the new one i.e., ' +
-                             path.join(common.tmpDir, '.node_repl_history') +
-                             ' and is empty.\nUsing it as is.\n';
+const replDisabled = tagLF`
+
+  Persistent history support disabled. Set the NODE_REPL_HISTORY environment
+  variable to a valid, user-writable path to enable.
+
+`;
+const convertMsg = tagLF`
+
+  Converting old JSON repl history to line-separated history.
+  The new repl history file can be found at ${
+  path.join(common.tmpDir, '.node_repl_history')}.
+
+`;
+const homedirErr = tagLF`
+
+  Error: Could not get the home directory.
+  REPL session history will not be persisted.
+
+`;
+const replFailedRead = tagLF`
+
+  Error: Could not open history file.
+  REPL session history will not be persisted.
+
+`;
+const sameHistoryFilePaths = tagLF`
+
```

Comment:
[suggestion] I know you don't like to change too much, but I would go
```js
const keepLF = '\n';
const fileName = path.join(common.tmpDir, '.node_repl_history');
const sameHistoryFilePaths = wrap`
  ${keepLF}
  The old repl history file has the same name and location as the new one i.e.,
  ${fileName} and is empty.
  Using it as is.
  ${keepLF}
`;
```
maybe even add `export.keepLF  = '\n'` to `common`